registry/commands.py

```diff
@@ -24,9 +24,11 @@ def create_user(email, password):
 
 @app.cli.command("install-test-data")
 def install_test_data():
-    with open("tests/data/imports.csv") as csv_file:
+    with open("tests/data/imports.csv", encoding="utf-8") as csv_file:
```

Comment:
Toto by mělo být správně. _imports.csv_ je v UTF-8 a to víme. _open_ oproti tomu ve výchozím režimu soubor otevírá textově s výchozím kódováním. To se může lišit podle platformy. Na českých Windows typicky _cp1250_, což logicky nechceme.